README.md

```diff
@@ -69,10 +69,13 @@ When `local_component_span` method is called, it creates a new span and a local
 
 Example:
 ```ruby
-ZipkinTracer::TraceClient.local_component_span('Local Trace') do |ztc|
-  ztc.record 'New Annotation'
-  ztc.record_tag 'key', 'sample'
-  # target process
+ZipkinTracer::TraceClient.local_component_span('DB process') do |ztc|
+  ztc.record 'Create users'
+  ztc.record_tag 'number', '1000'
+
+  ActiveRecord::Base.transaction do
+    1000.times { User.create(options) }
+  end
```

Comment:
I think this example serves no purpose but to make things more confusing and there are people out there not using ActiveRecord.
IMO it should be simply something like 
```ruby
# create 1000 users
```